CI fails

```
Cloning the remote Git repository

ERROR: Error cloning remote repo 'origin'

hudson.plugins.git.GitException: Command "git fetch --tags --force --progress -- https://github.com/ElektraInitiative/PermaplanT.git +refs/heads/*:refs/remotes/origin/*" returned status code 128:

stderr: fatal: unable to access 'https://github.com/ElektraInitiative/PermaplanT.git/': gnutls_handshake() failed: The TLS connection was non-properly terminated.
```

https://build.libelektra.org/blue/organizations/jenkins/PermaplanT/detail/master/500/pipeline

```
..RRFFFFRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRFRRFFFFFFFFFFFFFFFRRRRRR           [100%]R [100%]R [100%]R [100%]R [100%]R [100%]R [100%]F [100%]F [100%]F [100%]F [100%]F [100%]F [100%]

=================================== FAILURES ===================================

___________________________ test_map_already_exists ____________________________

[gw11] linux -- Python 3.10.6 /usr/bin/python3

```

--- a/ci/Jenkinsfile
+++ b/ci/Jenkinsfile
@@ -302,11 +302,11 @@ lock("${env.NODE_NAME}-exclusive") {
         node('docker') {
             node_info()
             checkout scm
-            if (env.BRANCH_NAME != 'master') {
-                echo "Skipping E2E Tests for branch ${env.BRANCH_NAME}"
+            def target = 'https://pr.permaplant.net'
+            if (env.BRANCH_NAME == 'master') {
+                echo 'Skipping E2E Tests for master'
                 return
             }
-            def target = 'https://pr.permaplant.net'
             try {
                 docker.build('permaplant-e2e:ci', './e2e').inside('-e E2E_URL=' + target) {
                     wait_for_pr_db()

--- a/ci/Jenkinsfile
+++ b/ci/Jenkinsfile
@@ -302,10 +302,11 @@ lock("${env.NODE_NAME}-exclusive") {
         node('docker') {
             node_info()
             checkout scm
-            def target = 'https://pr.permaplant.net'
-            if (env.BRANCH_NAME == 'master') {
-                target = 'https://dev.permaplant.net'
+            if (env.BRANCH_NAME != 'master') {
+                echo "Skipping E2E Tests for branch ${env.BRANCH_NAME}"
+                return
             }
+            def target = 'https://pr.permaplant.net'
             try {
                 docker.build('permaplant-e2e:ci', './e2e').inside('-e E2E_URL=' + target) {
                     wait_for_pr_db()

--- a/doc/changelog.md
+++ b/doc/changelog.md
@@ -59,7 +59,7 @@ Syntax: `- short text describing the change _(Your Name)_`
 - _()_
 - _()_
 - _()_
-- _()_
+- CI: disable E2E tests in master #1055 _(4ydan)_
 - _()_
 - _()_
 - _()_

